lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -515,7 +515,8 @@ module.exports = class MethodMappings {
         },
 
         setElementValueRedacted(...args) {
-          // FIXME: redact password in verbose HTTP logs, it's only redacted in the command logs
+          global._redactParams = true;
```

Comment:
This is the kind of hack I did, as I wasn't able to access `options.redact` anymore. As when we use `setPassword` command, `setElementValueRedacted` is called, and it again calls `setElementValue`, where we use `sendKeys(<value>)` to send value. Now, `selenium` create and build request out of it. After that it sends the request, where https://github.com/vaibhavsingh97/nightwatch/blob/f161aead566f1c12bed006973c6fea79784f18f5/lib/transport/selenium-webdriver/index.js#L143-L143, this function receive request which selenium had built, and we log this in https://github.com/vaibhavsingh97/nightwatch/blob/c06a66a9a5dc86c6932134e398c0d0493a80c13e/lib/http/request.js#L184-L184

Reply:
@beatfactor I wasn't able to figure it out, if you have any suggestions than let me know

Reply:
I think the hack is it's unavoidable in this case, but you can add the flag on to the nightwatch instance instead of global.

Reply:
Initially, I tried to add a flag in `NightwatchInstance`, but I can't access it when we log `requests`, any pointers for this?

Reply:
You can set the `globalSettings` like it's done [here](https://github.com/nightwatchjs/nightwatch/blob/main/lib/index.js#L390), only you have to modify the setter in http/request.js, to something like:

```js
 static set globalSettings(val) {
    __globalSettings__ = Object.assign({}, val);
  }
```

Reply:
@vaibhavsingh97 any updates here?

Reply:
Nope, need to work on it, I will update the PR today